pkg/kubelet/cm/cpumanager/cpu_manager.go

```diff
@@ -44,6 +44,9 @@ type runtimeService interface {
 
 type policyName string
 
+// DefaultCPUManagerStateFilePath is path to state file
+const DefaultCPUManagerStateFilePath = "/var/lib/kubelet/cpu_manager_state"
```

Comment:
this should be calculated relative to root directory.

https://github.com/kubernetes/kubernetes/blob/3a15fdbe7e1ceefa59d7cc41ac6d65998cef1b1a/cmd/kubelet/app/options/options.go#L93

can we update this?

Reply:
ok